content/components/avoid-functions-in-templates.md

```diff
@@ -0,0 +1,101 @@
+---
+title: avoid functions in templates
+---
+
+# Problem
+
+Since angular can not know if result of the function (or a getter) has changed, in default ChangeDetection mode the component will continuously execute template bound function to keep up-to-date with underlying data flow. 
```

Comment:
Some suggested rewording:
When you are using `ChangeDetection` in default mode, Angular is gonna check every function that is bound in the template of a Component over and over again. This is because Angular cannot know whether or not the result of that function call has changed.
This can cause performance issues!